wgsl_bindgen/src/wgsl_bindgen.rs

```diff
@@ -135,6 +135,9 @@ pub struct WgslBindgenOption {
   /// The additional set of directories to scan for source files.
   #[builder(default, setter(into, each(name = "additional_scan_dir", into)))]
   pub additional_scan_dirs: Vec<AdditionalScanDirectory>,
+
+  #[builder(default)]
+  pub capabilities: crate::Capabilities,
```

Comment:
I would rename this to `ir_capabilities` and change the type to `Option< WgslShaderIRCapabilities>`

Reply:
Renamed